main/res/values/strings.xml

```diff
@@ -1564,6 +1565,12 @@
     <string name="search_bookmark_list">Bookmark list</string>
     <string name="search_pocket_is_new">new</string>
     <string name="search_pocket_and_bookmark_loading">Loading a list of Pocket Queries and bookmark lists</string>
+    <string name="search_bookmark_loading">Loading bookmark lists</string>
+    <string name="search_bookmark_select">Select bookmark lists</string>
+    <string name="search_bookmark_new">Create new bookmark lists</string>
+    <string name="search_bookmark_adding_caches_success">All GC geocaches where added to the selected bookmark list</string>
```

Comment:
"were" instead of "where" or even better "have been"

Reply:
You're totally right. That happens when you focus more on code architecture, than strings... Solved as well.